subprojects/packagefiles/pugixml/meson.build

```diff
@@ -1,4 +1,4 @@
-project('pugixml', 'cpp', version: '1.12', meson_version: '>=0.46.0')
+project('pugixml', 'cpp', default_options : 'cpp_std=c++14', version: '1.12.1', meson_version: '>=0.46.0')
```

Comment:
this isn't working...

Reply:
That should work with meson master, but with meson 0.62 that option is ignored on subprojects.

Reply:
AFAIK meson 0.62.1 is used with CI:

```
2022-04-27T21:04:42.8479195Z Collecting meson
2022-04-27T21:04:42.8952087Z   Downloading meson-0.62.1-py3-none-any.whl (863 kB)
2022-04-27T21:04:43.0333993Z Installing collected packages: meson
2022-04-27T21:04:44.3168434Z Successfully installed meson-0.62.1
```